lib/tasks/projects.rake

```diff
@@ -135,4 +135,19 @@ namespace :projects do
     number_to_sync = args.number_to_sync || 2000
     Project.least_recently_updated_stats.where(platform: supported_platforms).limit(number_to_sync).each{|project| project.update_maintenance_stats_async(priority: :low)}
   end
+
+  desc 'Set license_normalized flag'
+  task set_license_normalized: :environment do
+    supported_platforms = ["Maven", "NPM", "Pypi", "Rubygems", "Nuget", "Packagist"]
```

Comment:
assuming these are the right capitalizations that libraries uses

Reply:
Need to fix up `NuGet`